src/components/IssuesList/IssuesList.tsx

```diff
@@ -7,17 +10,21 @@ import {
   Text,
   View,
 } from "react-native";
-import { RootStackParams, DataItem } from "../types";
-import { useIssuesContext } from "../contexts/IssuesContext";
-import IssuesItem from "./IssuesItem";
-import { useIssuesData } from "../hooks/useIssuesData";
-
-type Props = NativeStackScreenProps<RootStackParams, "Issues">;
-
-const IssuesList: FC<Props> = ({ navigation }) => {
-  const { issuesData, isLoading } = useIssuesData();
-  console.log("IssuesList", isLoading);
+import { RootStackParams, DataItem } from "../../types";
+import { IssuesItem } from "components/IssuesItem";
+import { RouteProp } from "@react-navigation/native";
 
+type IssuesListProp = {
+  navigation: NativeStackNavigationProp<RootStackParams, "Issues", undefined>;
+  route: RouteProp<RootStackParams, "Issues">;
+  isLoading: boolean;
+  issuesData: DataItem[];
+};
+const IssuesList: FC<IssuesListProp> = ({
+  navigation,
```

Comment:
pass down a function created by you that accepts an argument instead of passing `navigation`